srtcore/api.cpp

```diff
@@ -2980,7 +2973,7 @@ SRTSOCKET CUDT::accept(SRTSOCKET u, sockaddr* addr, int* addrlen)
       s_UDTUnited.setError(new CUDTException(e));
       return INVALID_SOCK;
    }
-   catch (bad_alloc&)
+   catch (const bad_alloc&)
```

Comment:
Adding const here is superfluous, provided there's no name declared here. It's kinda neutral and in this case changes nothing.

Reply:
In general, I agree that this is superfluous. This is reported by SonarCloud: [link](https://sonarcloud.io/project/issues?id=srt&issues=AXDFjdaG1BT7hYfjdXR3&open=AXDFjdaG1BT7hYfjdXR3). Although there is no variable declared, still I think nothing bad from extra const, that ensures proper casting.